ui/components/multichain/app-footer/app-footer.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { AppFooter } from '.';
+
+export default {
+  title: 'Components/Multichain/AppFooter',
+};
+// TODO: fix story alignment issues
```

Comment:
Can we have a dedicated ticket for this TODO?

Reply:
I managed to fix it in my most recent commit, just forgot to take out the comment. Thanks!